schema.sql

```diff
@@ -29,4 +29,10 @@ CREATE TABLE vets ( id INT GENERATED ALWAYS AS IDENTITY, name VARCHAR(250), age
 
 CREATE TABLE specializations(species_id INT, vets_id INT, PRIMARY KEY (species_id, vets_id), FOREIGN KEY(species_id) REFERENCES species (id), FOREIGN KEY (vets_id) REFERENCES vets (id));
 
-CREATE TABLE visits (animals_id INTEGER NOT NULL, vets_id INTEGER NOT NULL, visit_date DATE, FOREIGN KEY (animals_id) REFERENCES animals (id), FOREIGN KEY (vets_id) REFERENCES vets (id));
\ No newline at end of file
+CREATE TABLE visits (animals_id INTEGER NOT NULL, vets_id INTEGER NOT NULL, visit_date DATE, FOREIGN KEY (animals_id) REFERENCES animals (id), FOREIGN KEY (vets_id) REFERENCES vets (id));
+
+-- /*Project 5*/
+
+CREATE INDEX animal_id_index ON visits(animal_id);
+CREATE INDEX vet_id_index ON visits(vet_id);
+CREATE INDEX email_index ON owners(email);
```

Comment:
- [ ] I can see that you added an index and ran this query:

```sql
SELECT * FROM visits where vet_id = 2;
```
Even if the execution times difference before you added the index and after you added the index is very small in your case, I can clearly see that you had some fun with the `explain analyze` command and I am sure that in the future you will know how to start the investigation for some slow DB queries 👍

Reply:
Will we see something about this in future lessons? @Gambit142